spec/controllers/cargos_controller_spec.rb

```diff
@@ -11,12 +11,11 @@
   let!(:cargo2) { create :cargo, user_id: user.id, price: 2643 }
 
   context '#index' do
```

Comment:
Maybe we should add tests for multiply users

Reply:
multiple users?

Reply:
Yes. Does #index assign packages from other users?